src/containers/WishListsPage.less

```diff
@@ -50,6 +50,14 @@
   }
 
   .wish-list-container-inner {
+    .message-permission {
+      h2{
+        color: red;
+        text-align: center;
```

Comment:
Can we please also add some top padding so it's not flush with any divs before it
![image](https://user-images.githubusercontent.com/39800563/53823760-3b23ff00-3f27-11e9-9266-b1d8971c0018.png)